### DISCUSSION TITLE
MacOS Support

### DISCUSSION THREAD

**Discussion by The-MG** [maximmax42/Discord-CustomRP#51] [Ideas]
Add MacOS support for MacOS Intel processors and MacOS M1 processors.

**Comment by maximmax42** [comment]
Impossible without rewriting the whole app to a different framework which I don't plan on doing any time soon.

**Reply by maximmax42** [reply]
You can try using Wine, altho I have no idea if Macs even have Wine.

**Reply by cuboooooo** [reply]
I use wine on my Mac. I'm not sure if it would work with CustomRP however.
